install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -745,12 +745,12 @@ data:
 {{- end }}
 
 {{- if hasKey .Values "hostPort" }}
-{{- if or (eq $kubeProxyReplacement "partial") (eq $kubeProxyReplacement "false") }}
+{{- if eq $kubeProxyReplacement "partial" }}
```

Comment:
So `hostPort` and `externalIPs` Helm values take effect only if `kubeProxyReplacement` is set to `"partial"`, but partial kpr is deprecated for a while, right? The it might make sense to mark these Helm values as deprecated too?

Reply:
I believe typically Helm values are just described as deprecated in the Helm doc comment, like [here](https://github.com/cilium/cilium/blob/d594d1f47ff7035edb7635de8abf1a912bb660f9/install/kubernetes/cilium/values.yaml#L1197-L1199), plus upgrade notes. But I'm not sure if Helm templates should use "partial" kpr value if it's deprecated - I suppose it was kept for backwards compatibility, but maybe at this point it can be removed entirely?